### DISCUSSION TITLE
STUdio : Impossible de glisser déposer les packs d'histoires

### DISCUSSION THREAD

**Discussion by Fikk78** [marian-m12l/studio#477] [Q&A]
Bonjour à tous
Je n'arrive pas a avoir la petite main pour glisser deposer les packs sur la boite et inversement dans STUdio.
J'ai bien ma boite avec mes histoires qui apparait et j'ai bien des packs que j'avais réussi à télécharger.
Au lieu d'avoir la petite main j'ai un panneau d'interdiction comme sur la photo uniquement sur les pack notés "officiel".

![Capture d'écran 2024-08-30 115712](https://github.com/user-attachments/assets/f8192e07-f39e-4bec-a0e5-7307660a7cd5)

**Comment by marian-m12l** [comment]
Bonjour,
C'est bien le comportement attendu. Passez par l'application officielle pour gérer vos packs d'histoires officiels :)

**Reply by Fikk78** [reply]
Bonjour,

Merci de votre réponse.
Savez vous comment les transférer dans l’application officielle ? Car c’est des packs que j’ai téléchargé .

Cordialement

**Comment by Fikk78** [comment]
Bonjour,

Merci de votre réponse.
Savez vous comment les transférer dans l’application officielle ? Car c’est des packs que j’ai téléchargé .

Cordialement

Le 31 août 2024 à 22:37, marian-m12l ***@***.***> a écrit :

﻿

Bonjour,
C'est bien le comportement attendu. Passez par l'application officielle pour gérer vos packs d'histoires officiels :)
